hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/upgrade/OMLayoutFeature.java

```diff
@@ -44,7 +44,8 @@ public enum OMLayoutFeature implements LayoutFeature {
 
   FILESYSTEM_SNAPSHOT(5, "Ozone version supporting snapshot"),
 
-  QUOTA(6, "Ozone quota re-calculate");
+  QUOTA(6, "Ozone quota re-calculate"),
+  HBASE_SUPPORT(7, "Full support of hsync for HBase");
```

Comment:
Can you mention the lease recovery too in the description?

Reply:
+1 a description of what actually changed in the metadata would be helpful